src/measure/Measure.i

```diff
@@ -96,6 +96,71 @@
 };
 
 #if defined SWIGRUBY
+  %feature("director:except") {
```

Comment:
I also contemplated moving this up to a higher level. I would be fine with that, but I opted to keep it "local" to where the directors are defined. I thought there was a bit of logic in that. Either way is really ok.